AirCasting/Utils/DataFormatters.swift

```diff
@@ -0,0 +1,100 @@
+// Created by Lunar on 30/09/2021.
+//
+
+import Foundation
+
+enum DateFormatters {
+    
+    enum SessionDownloadService {
+        static let decoderDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum SessionUploadService {
+        static let encoderDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.timeZone = TimeZone.init(abbreviation: "UTC")
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum CreateSessionAPIService {
+        static let encoderDateFormatter: DateFormatter = {
+            let df = DateFormatter()
+            df.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            return df
+        }()
+    }
+    
+    enum AirBeam3Configurator {
+        static let USLocaleDateFormatter: DateFormatter = {
```

Comment:
I know `US` should be written in uppercase in "normal life", but in programming usually `camelCase` convention takes precedent so it actually should be `usLocaleDateFormatter`.